From ef9748b1c248b2f8c33ba08b6b01232142389476 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Thu, 31 Jan 2019 16:48:57 +0000
Subject: [PATCH 3/7] lineage: Rewrite Lineage Power HAL rules

 * Simplify the rules by labeling it with the already existing
   power-related hwservice context and move to /system to where
   it belongs.

Change-Id: I7340d6692671af71b1f79758775207f1d3955f43
---
 common/private/hwservice_contexts  | 1 +
 common/vendor/hal_power_default.te | 1 -
 common/vendor/hwservice.te         | 1 -
 common/vendor/hwservice_contexts   | 1 -
 common/vendor/system_server.te     | 1 -
 5 files changed, 1 insertion(+), 4 deletions(-)
 delete mode 100644 common/vendor/hal_power_default.te
 delete mode 100644 common/vendor/hwservice.te
 delete mode 100644 common/vendor/hwservice_contexts
 delete mode 100644 common/vendor/system_server.te

diff --git a/common/private/hwservice_contexts b/common/private/hwservice_contexts
index 2c5d9ce..ca6c2b6 100644
--- a/common/private/hwservice_contexts
+++ b/common/private/hwservice_contexts
@@ -1 +1,2 @@
 vendor.lineage.livedisplay::IColor                   u:object_r:hal_lineage_livedisplay_hwservice:s0
+vendor.lineage.power::ILineagePower                  u:object_r:hal_power_hwservice:s0
diff --git a/common/vendor/hal_power_default.te b/common/vendor/hal_power_default.te
deleted file mode 100644
index c693a2d..0000000
--- a/common/vendor/hal_power_default.te
+++ /dev/null
@@ -1 +0,0 @@
-allow hal_power_default hal_lineage_power_hwservice:hwservice_manager { add find };
diff --git a/common/vendor/hwservice.te b/common/vendor/hwservice.te
deleted file mode 100644
index 2ca8189..0000000
--- a/common/vendor/hwservice.te
+++ /dev/null
@@ -1 +0,0 @@
-type hal_lineage_power_hwservice, hwservice_manager_type;
diff --git a/common/vendor/hwservice_contexts b/common/vendor/hwservice_contexts
deleted file mode 100644
index 3ae7fd7..0000000
--- a/common/vendor/hwservice_contexts
+++ /dev/null
@@ -1 +0,0 @@
-vendor.lineage.power::ILineagePower      u:object_r:hal_lineage_power_hwservice:s0
diff --git a/common/vendor/system_server.te b/common/vendor/system_server.te
deleted file mode 100644
index 4a889fc..0000000
--- a/common/vendor/system_server.te
+++ /dev/null
@@ -1 +0,0 @@
-allow system_server hal_lineage_power_hwservice:hwservice_manager find;
-- 
2.17.1

